src/modules/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -56,7 +56,18 @@ function SuggestedMentionList(props: SuggestedMentionListProps): JSX.Element {
   const [searchString, setSearchString] = useState('');
   const [lastSearchString, setLastSearchString] = useState('');
   const [currentUser, setCurrentUser] = useState<User>(null);
-  const [currentMemberList, setCurrentMemberList] = useState<Array<Member>>([]);
+  const [currentMemberList, setCurrentMemberList] = useState<Member[]>([]);
+
+  const currentSearchedMemberList(): Member[] => useMemo(() => {
+    return currentChannel.members
+    .sort((a, b) => a.nickname?.localeCompare(b.nickname))
+    .filter(
+      (member) =>
+        member.nickname?.toLowerCase().startsWith(searchString.toLowerCase()) &&
+        member.userId !== currentUser?.userId &&
+        member.isActive,
+    ).slice(0, maxSuggestionCount);
+  }, [currentChannel.members.length])
```

Comment:
```currentChannel.members.map(member => member.nickname)```
will be better, I guess

Reply:
oh, it's my mistake, the other properties also being used
I think `currentChannel.members.length` is enough then... sorry;; @liamcho

Reply:
If you want to make it tight
```
currentChannel.members.map(member => member.nickname),
currentChannel.members.map(member => member.userId),
currentChannel.members.map(member => member.isActive),
```

This will work, It seems `member.state` is not used here

Reply:
I think length will do for userId one. I changed to isActive!